.github/workflows/ci.yml

```diff
@@ -33,46 +32,47 @@ jobs:
         toolchain: 1.70.0
     - name: Check format
       run: bash ./check.sh
+
   sbor-unit-tests:
     name: Run SBOR unit tests
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         os: [k8s-linux-runner]
     steps:
-    - uses: actions/checkout@v3
-    - uses: actions-rs/toolchain@v1
-      with:
-        toolchain: 1.70.0
```

Comment:
How is the rust version configured for code coverage tests?

Reply:
good question. so if you take a look at the current pipeline you'll see that there are usually 2 steps
1. setup rust
2. run tests with coverage
In step 1 you can set the toolchain version, which currently defaults to 1.70.0: https://github.com/radixdlt/public-iac-resuable-artifacts/blob/main/rust-setup/action.yml#L4-L7